packages/services/.gitignore

```diff
@@ -8,5 +8,3 @@ jspm_packages
 
 # Webpack directories
 .webpack
-
-.env
```

Comment:
Would this not be an extra safety measure?

Reply:
No, that's fine - we already have `.env*` rule in this file - it's doing all the job